app/src/main/java/com/pajato/android/gamechat/fragment/GameFragment.java

```diff
@@ -28,19 +31,26 @@
 import java.util.ArrayList;
 
 /**
- *
+ * A Fragment that contains and controls the current game being played.
  *
  * @author Bryan Scott
  */
 public class GameFragment extends BaseFragment{
+    /* individual fragment indicator numbers */
```

Comment:
Use JavaDoc comments: /** A short concise sentence. */ that uses punctuation and capitalization correctly.   A blank line between each commented variable is ok and might be helpful.